Fix Trigger Missions

EmosewaMC opened the pull request:
Fix trigger missions as previously they were all compared as lowercase, but the code was change to be uppercase.  To keep consistent with the value in the mission and the value in the lutriggers file

Tested that Discover the Orange Path, Cobwebs!, and Insect Music all progressed correctly (they are all trigger progression)

MustacheMan2 commented:
Are there currently known issues with these trigger missions? Having issues with the orange path and others on the latest commit.

EmosewaMC commented:
Yes, this addresses those issues with trigger missions.  The monument is weird where 1 of the paths uses triggers and the other 2 use POI. @MustacheMan2
